app/controllers/application_controller.rb

```diff
@@ -1,5 +1,15 @@
 class ApplicationController < ActionController::Base
-  # Prevent CSRF attacks by raising an exception.
-  # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  include ApplicationHelper
```

Comment:
@mayur-kambariya Eh, We should not include helper in the controller usually, unless we are using all the functions from it.